docs/get-started.md

```diff
@@ -11,13 +11,38 @@ weight=-70
 
 # Get started with Docker Machine and a local VM
 
-Let's take a look at using `docker-machine` for creating, using, and managing a
-Docker host inside of <a href=
-"https://www.virtualbox.org/" target="_blank">VirtualBox</a>.
+Let's take a look at using `docker-machine` to create, use and manage a
+Docker host inside of a local virtual machine.
+
+## Prerequisite Information
+
+With the advent of [Docker for Mac](/docker-for-mac/index.md) and [Docker for Windows](/docker-for-windows/index.md) as replacements for [Docker Toolbox](/toolbox/overview.md), there are some new considerations for creating local machines with `docker-machine create`.
+
+The new solutions come with their own native virtualization solutions rather than using Oracle VirtualBox. In the case of Docker for Windows, you'll use a different driver to create machines.
```

Comment:
Hm, we might want to consider something like

> "Docker for Mac allows for the creation of additional `docker-machine`-based VirtualBox machines without issue, but if you'd like to use Docker for Windows side-by-side with Machine you will need to use the `hyper-v` driver instead."

I didn't understand what this paragraph was going for first time I read it.